.github/workflows/build.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        api-level: [16, 23, 29]
+        api-level: [21, 23, 29]
```

Comment:
Maybe have one with the highest we support instead of 29? (Or is that still 29?)

Reply:
Yes I though of the same thing, but I though we could maybe wait to do this with #853, and then do something like [21, 29, 33].

Reply:
My understanding is that these are the API levels of emulators that we want to run the tests against on GitHub workflow. We could update it

Reply:
[21, 29, 33] would be good to test the minSdk, something in between, and the max

Reply:
> So this PR can be merged as it is?

For me, yes.

> My understanding is that these are the API levels of emulators that we want to run the tests against on GitHub workflow. We could update it

Exactly. Maybe to the minimum API, the target API and some interesting API level in between where things could break. I don't know which one would be a good pick as the third one.

Reply:
I'd do [21, 29, 31] then? since it's our target API

Reply:
> and some interesting API level in between where things could break.

Good idea, but I don't know which one :)